xds/internal/xdsclient/filter_chain.go

```diff
@@ -391,13 +410,19 @@ func (fci *FilterChainManager) addFilterChainsForSourcePorts(srcEntry *sourcePre
 }
 
 // filterChainFromProto extracts the relevant information from the FilterChain
-// proto and stores it in our internal representation.
-func filterChainFromProto(fc *v3listenerpb.FilterChain) (*FilterChain, error) {
-	httpFilters, err := processNetworkFilters(fc.GetFilters())
+// proto and stores it in our internal representation. It also persists any
+// RouteNames which need to be queried dynamically via RDS.
+func (fci *FilterChainManager) filterChainFromProto(fc *v3listenerpb.FilterChain) (*FilterChain, error) {
+	filterChain, err := processNetworkFilters(fc.GetFilters())
 	if err != nil {
 		return nil, err
 	}
-	filterChain := &FilterChain{HTTPFilters: httpFilters}
+	// These Route Names will be dynamically queried via RDS in the wrapped
```

Comment:
Nit:
s/Route Names/route names/
s/Filter Chain/filter chain/

`FilterChain` or `RouteConfigName` could work as well. But `Filter Chain` and `Route Name` seems to make the comment stutter a bit, while also not being as helpful as they can otherwise be.

Reply:
Switched.